acc: Skip utf-8 check for main output file

denik opened the pull request:
## Why
This check originally was added to prevent printing large binary files (e.g. zip files). However, "bundle deploy --debug" can output binary data as well (request and response bodies), causing test failure and not showing the contents, preventing debugging.

shreyas-goenka reviewed (approved):
Should we get rid of this check entirely? Suppose I redirect the debug output to another file, that would not apply replacements and would be really hard to debug.

denik commented:
> Should we get rid of this check entirely? Suppose I redirect the debug output to another file, that would not apply replacements and would be really hard to debug.

For context, I added it when I was working with zip/whl files and it was annoying when they were printed / diffed / copied so that I added this check. I think in general the check is still valid unless we decide we want binary files in test fixtures but then we need to add better support for them.

Related, I think it's "--debug" that should be fixed to escape or skip or decode binary output before printing. This is something on SDK side though cc @renaudhartert-db @pietern
